share/pedersen/vss/vss.go

```diff
@@ -402,11 +402,20 @@ func (v *Verifier) decryptDeal(e *EncryptedDeal) (*Deal, error) {
 	return deal, err
 }
 
+// ErrNoDealBeforeResponse is an error happening if the verifiers receives a
```

Comment:
English fixes and removing the reference to DKG, which is not really relevant here:

ErrNoDealBeforeResponse is an error returned if a verifier receives a
deal before having received any responses. For the moment, the caller must
be sure to have dispatched a deal before.